openmdao/docs/advanced_guide/derivs/partial_derivs_explicit.rst

```diff
@@ -22,7 +22,7 @@ Here is an example, based on the :ref:`Betz Limit Example <betz_limit_tutorial>`
 
 
 The calls to :code:`declare_partials` tell OpenMDAO which partial derivatives to expect.
-This is always done inside the :code:`setup` method.
+This should always be done inside the :code:`setup_partials` method.
```

Comment:
`should` or `must`?
if `should` then more explanation is probably required...
if `must` then we do we need a deprecation or exception when it's done the old way?

Reply:
I added more explanation about why setup_partials is the recommended way.